distribution/lib/Standard/Table/0.0.0-dev/src/Data/Group_By.enso

```diff
@@ -0,0 +1,25 @@
+from Standard.Base import all
+
+## Group By Key
+type Group_By_Key
+    type Group_By_Key values
+
+    ## See if two keys are equal
+    == : Group_By_Key->Boolean
+    == that =
+        if that.values.length != this.values.length then False else
+            0.up_to this.values.length . all i->(this.values.unsafe_at i == that.values.unsafe_at i)
+
+    ## See if two keys are not equal
+    != : Group_By_Key->Boolean
+    != that = this==that . not
```

Comment:
Again, this is already automatically handled by the definition at `Any.!=`. No need to repeat it (or were there issues without this?? Then I think it would be a bug worth reporting)